Decred Wallet/Wallet Utils/PoliteiaNotification.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PoliteiaNotification.swift
+//  Decred Wallet
+//
+// Copyright © 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import Foundation
+import Dcrlibwallet
+
+class PoliteiaNotification: NSObject {
+    static let shared = PoliteiaNotification()
+    
+    func syncPoliteia() {
+        DispatchQueue.global(qos: .userInitiated).async {
+            do {
+                try WalletLoader.shared.multiWallet.politeia?.sync(self)
+            } catch let error {
+                print("PoliteiaNotification sync Error:", error.localizedDescription)
```

Comment:
This error should be handled properly, display the error message to the user using the banner, there should be a retry button somewhere in the app also.

Reply:
I think it should only display the message: "There was an error when sync politeia and politeia would be synced the next time"